spec/Section 2 -- Language.md

```diff
@@ -1215,13 +1215,70 @@ size `60`:
 
 **Variable Use Within Fragments**
 
-Variables can be used within fragments. Variables have global scope with a given
-operation, so a variable used within a fragment must be declared in any
-top-level operation that transitively consumes that fragment. If a variable is
-referenced in a fragment and is included by an operation that does not define
-that variable, that operation is invalid (see
+Variables can be used within fragments. Operation-defined variables have global
+scope with a given operation, so a variable used within a fragment must either
+be declared in any top-level operation that transitively consumes that fragment,
```

Comment:
The sentence would flow better if we do something like this --

Operation-defined variables have... Fragment-defined variables have ... So a variable used within a fragment must ...

Thoughts?

Reply:
That sounds great, what do you think about https://github.com/graphql/graphql-spec/pull/1081/commits/5a7354371b14a3fcc42b5b376217283669869396